src/ngraph/runtime/cpu/pass/cpu_assignment.cpp

```diff
@@ -728,8 +728,14 @@ namespace ngraph
                 void CPUAssignment::ASSIGN_DECL(ngraph::op::Dequantize)
                 {
                     auto dequantize = static_cast<op::Dequantize*>(node);
-                    auto offset_const_op =
-                        std::static_pointer_cast<ngraph::op::Constant>(dequantize->get_argument(2));
+                    auto offset_const_op = std::dynamic_pointer_cast<ngraph::op::Constant>(
+                        dequantize->get_argument(2));
```

Comment:
use is_constant instead of dynamic casting.

Reply:
I use that to read the value of offset as well if its a constant. Not casting just for the check

Reply:
dynamic_casts dont work as expected on Mac. So check for is_constant and then do a static_cast.